server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -541,7 +541,7 @@ public void vaultExportShouldExportZipContainsOnlyMatchedMessages() throws Excep
         String fileLocation = exportAndGetFileLocationFromLastMail(exportRequest, bartAccessToken);
 
         try (ZipAssert zipAssert = assertThatZip(fileSystem.getResource(fileLocation))) {
-            zipAssert.containsOnlyEntriesMatching(hasName(firstMessageIdOfHomer));
+            zipAssert.containsOnlyEntriesMatching(hasNameContains(firstMessageIdOfHomer));
```

Comment:
`zipAssert.containsOnlyEntriesMatching(hasNameContains(firstMessageIdOfHomer + ".eml"));`

Would be more valuable and would not require this `hasNameContains` testing method, no?